fe2-android/app/src/androidTest/java/com/github/dedis/popstellar/ui/detail/event/fragments/ElectionSetupFragmentTest.java

```diff
@@ -0,0 +1,232 @@
+package com.github.dedis.popstellar.ui.detail.event.fragments;
+
+import static androidx.test.espresso.action.ViewActions.click;
+import static androidx.test.espresso.assertion.ViewAssertions.matches;
+import static androidx.test.espresso.matcher.ViewMatchers.isDisplayed;
+import static androidx.test.espresso.matcher.ViewMatchers.withText;
+import static com.github.dedis.popstellar.pages.detail.event.EventCreationPageObject.datePicker;
+import static com.github.dedis.popstellar.pages.detail.event.EventCreationPageObject.endDateView;
+import static com.github.dedis.popstellar.pages.detail.event.EventCreationPageObject.endTimeView;
+import static com.github.dedis.popstellar.pages.detail.event.EventCreationPageObject.pickerAcceptButton;
+import static com.github.dedis.popstellar.pages.detail.event.EventCreationPageObject.startDateView;
+import static com.github.dedis.popstellar.pages.detail.event.EventCreationPageObject.startTimeView;
+import static com.github.dedis.popstellar.pages.detail.event.EventCreationPageObject.timePicker;
+
+import android.icu.util.Calendar;
+
+import androidx.test.espresso.contrib.PickerActions;
+import androidx.test.filters.LargeTest;
+import androidx.test.internal.runner.junit4.AndroidJUnit4ClassRunner;
+
+import com.github.dedis.popstellar.testutils.FragmentScenarioRule;
+import com.github.dedis.popstellar.ui.detail.event.election.fragments.ElectionSetupFragment;
+
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+@LargeTest
+@RunWith(AndroidJUnit4ClassRunner.class)
+public class ElectionSetupFragmentTest {
+
+  private static final int YEAR;
+  private static final int MONTH_OF_YEAR;
+  private static final int DAY_OF_MONTH;
+  private static final String DATE;
+
+  private static final int HOURS = 12;
+  private static final int MINUTES = 15;
+  private static final String TIME = "" + HOURS + ":" + MINUTES;
+
+  static {
+    // Make sure the date is always in the future
+    Calendar today = Calendar.getInstance();
+    today.add(Calendar.MONTH, 13);
+
+    YEAR = today.get(Calendar.YEAR);
+    MONTH_OF_YEAR = today.get(Calendar.MONTH);
```

Comment:
Calendar.MONTH start at 0
```suggestion
    MONTH_OF_YEAR = today.get(Calendar.MONTH) + 1;
```

Reply:
done